SmplEditor/MainWindow.xaml.cs

```diff
@@ -338,39 +397,48 @@ private void OnSortOptionChanged(object sender, SelectionChangedEventArgs e)
             int selectedOption = SortOptionComboBox.SelectedIndex;
             switch (selectedOption){
                 case 0: // by Order, directory
+                // current structure of <Song> has a problem in this
```

Comment:
This have been fixed in [Sort by custom ordering in SMPL](https://github.com/Mins0o/Smpl-Editor/commit/50379c08ef8a9fe6862a81b39191f1702d218afb), 50379c0